### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by ohadbitt**
Just for sdk and docs

### CODE REVIEW

File: docs/KustoSource.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ that is using it. Please verify the following before using Kusto connector:
  
  Kusto connector implements Spark 'Datasource V1' API. 
  Kusto data source identifier is "com.microsoft.kusto.spark.datasource". 
- Kusto table schema is translated into a spark as explained in [DataTypes](Spark-Kusto DataTypes mapping.md).
+ Kusto table schema is translated into a spark as explained in [Spark-Kusto DataTypes](mapping.md).
```
Comment (danield137) [inline docs/KustoSource.md]: >  into a spark 
what does this mean?

File: docs/KustoSource.md [hunk 2]
```diff
@@ -35,15 +35,16 @@ that is using it. Please verify the following before using Kusto connector:
...
   ```
  where:
  * **Kusto-query** is any valid Kusto query. For details, please refer to [Query statements documentation](https://docs.microsoft.com/en-us/azure/kusto/query/statements). To read the whole table, just provide the table name as query.
  >Note:
  Kusto commands  cannot be executed via the connector.
  * **Parameters map** is a set of key-value pairs, where the key is the parameter name. See [Supported Options](#supported-options)
  section for details
- 
+ * **ClientRequestProperties** is a json of [ClientRequestProperties](https://github.com/Azure/azure-kusto-java/blob/master/data/src/main/java/com/microsoft/azure/kusto/data/ClientRequestProperties.java) object - 
+ optional. Parameters described [here](https://docs.microsoft.com/en-us/azure/kusto/api/netfx/request-properties).
```
Comment (danield137) [inline docs/KustoSource.md]: better to start with saying it's optional, then a short desc, then a link

File: docs/KustoSource.md (hunk 1 above)
Comment (ohadbitt) [reply docs/KustoSource.md]: nothing

Comment (danield137) [reply docs/KustoSource.md]: hmm was it supposed to say `into a spark scheme`?

File: docs/KustoSource.md (hunk 2 above)
Comment (ohadbitt) [reply docs/KustoSource.md]: this is actually a copy paste mistake

**Pull request merged**